Fix crash when extensions have Addons.xcu with nonstd merge points

diff --git a/framework/source/uielement/menubarmerger.cxx b/framework/source/uielement/menubarmerger.cxx
index 9e497b6..7067f96 100644
--- framework/source/uielement/menubarmerger.cxx
+++ framework/source/uielement/menubarmerger.cxx
@@ -104,6 +104,14 @@ ReferencePathInfo MenuBarMerger::FindReferencePath(
 {
     sal_uInt32       i( 0 );
     const sal_uInt32 nCount( rReferencePath.size() );
+
+    ReferencePathInfo aResult;
+    if ( !nCount )
+    {
+        aResult.eResult = RP_MENUITEM_NOT_FOUND;
+        return aResult;
+    }
+
     Menu*            pCurrMenu( pMenu );
     RPResultInfo     eResult( RP_OK );
 
@@ -145,7 +153,6 @@ ReferencePathInfo MenuBarMerger::FindReferencePath(
     }
     while (( pCurrMenu != 0 ) && ( i < nCount ) && ( eResult == RP_OK ));
 
-    ReferencePathInfo aResult;
     aResult.pPopupMenu = pCurrMenu;
     aResult.nPos       = nPos;
     aResult.nLevel     = nLevel;
